src/ts/greedy-nav/GreedyNav.ts

```diff
@@ -0,0 +1,1062 @@
+        el.className = `${el.className} ${className}`;
+    }
+};
+
+/**
+ * Show/hide toggle button
+ */
+export const showToggle = (
+    navWrapperElement: HTMLElement,
+    navDropdownToggleSelector: string,
+    breaks: number[]
+): void => {
+    if (breaks.length < 1) {
+        const navDropdownToggle = navWrapperElement.querySelector<HTMLElement>(
+            navDropdownToggleSelector
+        );
+
+        if (navDropdownToggle == null) {
+            return;
+        }
+
+        navDropdownToggle.classList.add('priority-nav-is-hidden');
+        navDropdownToggle.classList.remove('priority-nav-is-visible');
+        navWrapperElement.classList.remove('priority-nav-has-dropdown');
+
+        /**
+         * Set aria attributes for accessibility
+         */
+
+        const navWrapper = navWrapperElement.querySelector<HTMLElement>(
+            '.priority-nav__wrapper'
+        );
+
+        if (navWrapper) {
+            navWrapper.setAttribute('aria-haspopup', 'false');
+        }
+    } else {
+        const navDropdownToggle = navWrapperElement.querySelector<HTMLElement>(
+            navDropdownToggleSelector
+        );
+
+        if (navDropdownToggle == null) {
+            return;
+        }
+        navDropdownToggle.classList.add('priority-nav-is-visible');
+        navDropdownToggle.classList.remove('priority-nav-is-hidden');
+        navWrapperElement.classList.add('priority-nav-has-dropdown');
+
+        /**
+         * Set aria attributes for accessibility
+         */
+        const navWrapper = navWrapperElement.querySelector<HTMLElement>(
+            '.priority-nav__wrapper'
+        );
+
+        if (navWrapper) {
+            navWrapper.setAttribute('aria-haspopup', 'true');
+        }
+    }
+};
+
+/**
+ * Update count on dropdown toggle button
+ */
+const updateCount = (
+    _this: HTMLElement,
+    navDropdownToggleSelector: string,
+    breaks: number[]
+): void => {
+    // eslint-disable-next-line no-unused-expressions
+    _this
+        .querySelector<HTMLElement>(navDropdownToggleSelector)
+        ?.setAttribute('priorityNav-count', `${breaks.length}`);
+};
+
+export const updateLabel = (
+    menu: HTMLElement,
+    label: string,
+    navDropdownToggleSelector: string,
+    navDropdownLabelActive: string
+): void => {
+    const toggle = menu.querySelector<HTMLElement>(navDropdownToggleSelector);
+
+    if (toggle == null) {
+        return;
+    }
+
+    toggle.innerHTML = label;
+
+    if (label === navDropdownLabelActive) {
+        toggle.setAttribute('aria-expanded', 'true');
+    } else {
+        toggle.setAttribute('aria-expanded', 'false');
+    }
+};
+
+export const insertAfter = (newNode: Node, referenceNode: Node): void => {
+    if (referenceNode.parentNode !== null) {
+        referenceNode.parentNode.insertBefore(
+            newNode,
+            referenceNode.nextSibling
+        );
+    }
+};
+
+const checkForSymbols = (str: string) => {
```

Comment:
Again, why is this not a method on the Class?

Reply:
See above.